test/switch.test.js

```diff
@@ -49,6 +49,7 @@ it("ignores mixed children", () => {
     <Switch>
       Here is a<Route path="/users">route</Route>
       route
+      {null}
     </Switch>
   );
```

Comment:
Can we also add a separate test case "ignores falsey children" and test `false`, `null`, `undefined` etc.?

Reply:
Sure thing, I've done that now 😄 https://github.com/molefrog/wouter/pull/212/commits/1b5ad2fb2e4783bc80faf0276e656553cdb24f27